docs/UserGuide.md

```diff
@@ -28,21 +28,21 @@ This guide smooths the learning curve and helps you learn how to use _MyInsuRec_
 
 Before we go on further, look out for these symbols in the User Guide as they provide some important messages you might want to know!
 
-| Symbol               | Meaning                                                                 |
-|----------------------|-------------------------------------------------------------------------|
-| `command`            | Indicates some command or parameter that can be entered into MyInsuRec. |
-| :information_source: | Additional information.                                                 |
-| :exclamation:        | Crucial information needed for proper use of MyInsuRec.                 |
-| :bulb:               | Tips and tricks.                                                        |
+| Symbol               | Meaning                                                                                                                                      |
+|----------------------|----------------------------------------------------------------------------------------------------------------------------------------------|
+| `command`            | Indicates some command or parameter that can be entered into MyInsuRec.                                                                      |
+| :information_source: | Additional information. This symbol is followed by a label 'Note', and the background is blue in color.                                      |
+| :exclamation:        | Crucial information needed for proper use of MyInsuRec. This symbol is followed by a label 'Caution', and the background is yellow in color. |
+| :bulb:               | Tips and tricks. This symbol is followed by a label 'Tips and tricks', and the background is green in color.                                 |
```

Comment:
Do you think it is better to create 2 more columns for this? one for the label and one for the background color, rather than placing them all in one line.

Reply:
actually yes that wld be better, thanks for the suggestion